src/context/runTest.js

```diff
@@ -4,8 +4,10 @@ const headers = { 'Content-Type': 'application/json' }
 
 export default function runTest(snapshot, update) {
   const {
-    host = 'localhost',
-    port = 3000,
+    serverInfo: {
+      host = 'localhost',
```

Comment:
Good idea! I fear though that without webpack `DefinePlugin` defining deep-object env variables could be problematic... I've to try it.
An other possible solution could be to use [jest globals](http://facebook.github.io/jest/docs/configuration.html#globals-object) to create the `process.env.SNAPGUIDIST` object.